src/pages/search.js

```diff
@@ -33,7 +33,74 @@ class Search extends React.Component {
   script.setAttribute("defer", true)
 
   document.body.appendChild(script)
+*/
+    const script2 = document.createElement("script") // Loads the Addsearch JS blob from them
+    script2.setAttribute(
+      "src",
+      `https://cdn.jsdelivr.net/npm/addsearch-js-client@0.8/dist/addsearch-js-client.min.js`
+    )
+    //script2.setAttribute("defer", true)
+    document.body.appendChild(script2)
+
+
+    const script3 = document.createElement("script") // Loads the Addsearch JS blob from them
+    script3.setAttribute(
+      "src",
+      `https://cdn.jsdelivr.net/npm/addsearch-search-ui@0.7/dist/addsearch-search-ui.min.js`
+    )
+   // script3.setAttribute("defer", true)
+    script3.onload = () => this.addSearchStuff();
+   document.body.appendChild(script3)
+
+
+    const link = document.createElement("script") // Loads the Addsearch JS blob from them
+    link.setAttribute(
+      "href",
+      `https://cdn.jsdelivr.net/npm/addsearch-search-ui@0.7/dist/addsearch-search-ui.min.css`
+    )
+    link.setAttribute("rel", 'stylesheet')
+
+    //alert('mount up');
+    document.body.appendChild(link)
+
+
+
+
+
+
 }
+ addSearchStuff() {
```

Comment:
Can you chose a more meaningful method name? I wrote this name before I knew what it would do.